lib/hdcontroller.js

```diff
@@ -65,7 +80,10 @@ class HDProxydClient {
         this.bootstrap = (0, shuffle_1.shuffle)(this.bootstrap);
         this.path = '/store/';
         this.setCurrentBootstrap(this.bootstrap[0]);
-        this.httpAgent = new httpagent_1.http.Agent();
+        this.httpAgent = new httpagent_1.http.Agent({
+            freeSocketTimeout: 60 * 1000,
+            timeout: 2 * 60 * 1000,
```

Comment:
that [=the computation for timeout] is the default already, no?

Reply:
Yes, but I prefer it to be explicit, as httpagent is based on agentkeepalive, and it may not be clear if we rely on default values.